pkg/config/gateway/gateway_test.go

```diff
@@ -39,7 +39,7 @@ func TestIsTLSServerForNonHTTP(t *testing.T) {
 				},
 				Tls: &v1alpha3.ServerTLSSettings{HttpsRedirect: true},
 			},
-			expected: true,
+			expected: false,
```

Comment:
Why is this changed? Was it a bug earlier?

Reply:
Prvious commit changed it accidentally. And this is a revert